Assets/Scripts/Game/TalkManager.cs

```diff
@@ -2778,16 +2771,27 @@ private void AssembleTopiclistTellMeAbout()
                             break;
                         case QuestInfoResourceType.Location:
                             itemQuestTopic.questionType = QuestionType.QuestLocation;
-                            Place place = (Place)questResourceInfo.Value.questResource;
+                            Place place = (Place) questResourceInfo.Value.questResource;
```

Comment:
There shouldn't be a space here.

Reply:
Visual Studio automatically put this space, no pb I'll remove it. :)